src/App.tsx

```diff
@@ -1,56 +1,70 @@
-import React from 'react';
+import React, { useMemo, useRef, useState } from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import { DropdownMenu } from './DropdownMenu/DropdownMenu';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [query, setQuery] = useState('');
+  const [focus, setFocus] = useState(false);
```

Comment:
We at this state want to find out if there is currently a focus on the input, so it is better to change the naming a little
```suggestion
  const [isFocused, setIsFocused] = useState(false);
```